frontend/src/component/feature/FeatureView/FeatureView.tsx

```diff
@@ -145,6 +146,9 @@ export const FeatureView = () => {
     const [openStaleDialog, setOpenStaleDialog] = useState(false);
     const [isFeatureNameCopied, setIsFeatureNameCopied] = useState(false);
     const smallScreen = useMediaQuery(`(max-width:${500}px)`);
+    const hideFeatureEnvironmentVariants = useUiFlag(
```

Comment:
another option is to to a flag called show/display

Reply:
I think the name should be driven by what our desired value should be if we are unable to evaluate it from the feature flag system (I'm case it's down or the flag is accidentally removed). 

In the scenario of sunseting a feature the default action should be to not display the feature. With that rationale the active action would be to enable the feature, which should only be possible if the flag is correctly set. I would cast my vote for "enableLegacyFeatureVariant" to make it clear that you actively turn on a deprecated feature. 

(You could argue that hideX with a default value of true serves the same purpose and does look somewhat like a reversed kill switch. Personally it feels a bit twisted in my head)